navigator/backend/src/main/scala/com/digitalasset/navigator/store/platform/PlatformStore.scala

```diff
@@ -159,21 +159,34 @@ class PlatformStore(
   }
 
   def connected(state: StateConnected): Receive = {
-    case UpdateParties =>
+    case UpdatePartiesAndUsers =>
       state.ledgerClient.partyManagementClient
         .listKnownParties()
         .map(UpdatedParties(_))
         .pipeTo(self)
+
+      state.ledgerClient.userManagementClient
+        .listUsers() // TODO what token should we pass?
+        .map(UpdatedUsers(_))
+        .pipeTo(self)
       ()
+
+    case UpdatedUsers(users) =>
+      // TODO: should we be able to log in as a user without a primary party?
```

Comment:
For the trigger runner I chose to fail if you try to run a trigger as a user but the user has no primary party. So omitting them here seems reasonable as well imho.